include/drivers/flash.h

```diff
@@ -296,6 +307,25 @@ static inline size_t z_impl_flash_get_write_block_size(struct device *dev)
 	return api->write_block_size;
 }
 
+
+/**
+ *  @brief  Get pointer to flash_parameters structure
+ *
+ *  @return pointer to flash_parameters structure characteristic for
```

Comment:
... or NULL if parameters are not available.

Reply:
No NULL, withdrawing from making this optional.